app/forms/update_user_password_form.rb

```diff
@@ -6,9 +6,10 @@ class UpdateUserPasswordForm
 
   delegate :personal_key, to: :encryptor
 
-  def initialize(user, user_session = nil)
+  def initialize(user, user_session = nil, required_password_change = false)
```

Comment:
Feels like this could be a good opportunity to switch these to keyword arguments, since positional arguments are hard to follow out of context (e.g. `UpdateUesrPasswordForm.new(user, nil, false)`)

```suggestion
  def initialize(user:, user_session: nil, required_password_change: false)
```